tmt/result.py

```diff
@@ -269,7 +277,9 @@ def from_test_invocation(
             result: ResultOutcome,
             note: Optional[str] = None,
             ids: Optional[ResultIds] = None,
-            log: Optional[list[Path]] = None) -> 'Result':
+            log: Optional[list[Path]] = None,
+            subresult: Optional[list[SubResult]] = None,
+            ) -> 'Result':
```

Comment:
(Formatting nitpicking `ruff` might make obsolete): `) -> 'Result':` belongs on the same line as `subresult`.

Reply:
Modified, thanks.